articles/toc.yml

```diff
@@ -1,4 +1,10 @@
 - name: Getting Started
   items:
-    - name: Example
-      href: Example.md
\ No newline at end of file
+    - name: Downloading the XRTK
+      href: 00-DownloadingTheXRTK.md
```

Comment:
how come these have numerical prefixes?

Reply:
Just a formatting mechanism for the file, so we always have an order to documents for reading.